.github/workflows/coredns.yml

```diff
@@ -18,7 +21,7 @@ jobs:
 
     - uses: actions/setup-go@v5
       with:
-        go-version: "${{ steps.versions.outputs.golang }}"
+        go-version: "${{ env.GOLANG_VERSION }}"
```

Comment:
What is the reason we chose to move back to static versions vs using the one in the `.tool-versions` file? Feels like a regression.

Reply:
agree, it is a regress, it was more about aligning w/ implementations in the rest of the repos, but .tool-versions version as base is cleaner. I will revert this. thanks for the feedback.